src/scripts/main.js

```diff
@@ -1,3 +1,21 @@
 'use strict';
 
-// write your code here
+const population = document.querySelectorAll('.population');
+
+const content = Array.from(population).map(element => element.textContent);
+
+const contentNew = [];
+
+for (let i = 0; i < content.length; i++) {
+  contentNew.push(+content[i].replace(/,/g, ''));
```

Comment:
Maybe instead of `contentNew` name can be more descriptive like `populationArray` ?